### PR TITLE
VIH-7768 adding hearing venue to required responses

### PR DESCRIPTION
**Pull request by omio-ma**
### JIRA link ###
https://tools.hmcts.net/jira/browse/VIH-7768

### Change description ###
- updating contract response to include scottish venue flag that will be used by video web to switch logos

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: VideoApi/VideoApi.Contract/Consts/HearingVenueNames.cs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+namespace VideoApi.Contract.Consts
+{
+    public static class HearingVenueNames
```
Comment (MichaelKeaneathmcts) [inline VideoApi/VideoApi.Contract/Consts/HearingVenueNames.cs:3]: Name it as ScottishHearingVenueNames ?

Comment (omio-ma) [reply VideoApi/VideoApi.Contract/Consts/HearingVenueNames.cs:3]: i left this generic so later if we need to add more venue names const we can use this cs file

Comment (MichaelKeaneathmcts) [reply VideoApi/VideoApi.Contract/Consts/HearingVenueNames.cs:3]: That will work for now.  I'm thinking more for the future when there are a lot of venues and possibly venue properties too.

**Review by robertbennett1998 — APPROVED** [review]
LGTM! Nice work

**Review by ZJianWang — APPROVED** [review]

**Review by PrasannaDommalapati — APPROVED** [review]
LGTM

**Pull request merged**